client/main/analyses.js

```diff
@@ -103,6 +103,9 @@ const Analyses = Backbone.Model.extend({
     defaults : {
         dataSetModel : null
     },
+    analysesLoaded: function() {
```

Comment:
maybe use count() rather than analysesLoaded() ... you'll need to iterate over the analyses and not count those that have been deleted.